packages/terra-action-footer/docs/README.md

```diff
@@ -0,0 +1,45 @@
+# Terra Action Footer
+
+The terra-action-footer component is a footer bar that contains two sockets, start and end, to place items in the form of buttons and hyperlinks used to perform actions. If no actions are provided, the footer bar defaults to a 10px equivalent height and maintains the top border.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-action-footer`
+
+## Usage
+
+### Action Footer with a Start and End Actions
+```jsx
+import React from 'react';
+import ActionFooter from 'terra-action-footer';
+import Button from 'terra-button';
+
+<ActionFooter
+  start={<a href="/link/to/somewhere">A link</a>}
+  end={
+    <div>
+      <Button text="Left End Button"/>
```

Comment:
Can we use terms other than `Left` and `Right`?

Won't the order of the buttons, within the `end` socket reverse?

In the component's readme as well

Reply:
In item view we use the terms 'start' and 'end' for our accessories to avoid this problem.
http://engineering.cerner.com/terra-clinical/#/site/components/clinical-site/item-view/index